lua/nvim-treesitter-textobjects.lua

```diff
@@ -28,7 +28,15 @@ function M.init()
           return M.has_textobjects(lang) or has_some_textobject_mapping(lang)
         end,
         lookahead = false,
-        keymaps = {}
+        keymaps = {},
+        selection_modes = {
```

Comment:
Think this should be empty and always default to `v`? Otherwise, we would need to update this every time a new one is added.

Reply:
Good point @stsewd, fixed in 7c71f39